src/interactive.ts

```diff
@@ -49,19 +67,35 @@ async function askStarterName(): Promise<string> {
   return starterName;
 }
 
-function getChoices() {
-  const maxLength = Math.max(...STARTERS.map(s => s.name.length)) + 1;
+/**
+ * Generate a terminal-friendly list of options for the user to select from
+ * @returns a formatted list of starter options
+ */
+function getChoices(): ReadonlyArray<{title: string, value: string}> {
+  const maxLength = Math.max(...STARTERS.map(s => generateStarterName(s).length)) + 1;
   return [
-    ...STARTERS.filter(s => s.hidden !== true).map(s => {
-      const description = s.description ? dim(s.description) : '';
-      return {
-        title: `${padEnd(s.name, maxLength)}   ${description}`,
-        value: s.name,
-      };
-    }),
+    ...STARTERS
+      .filter(s => s.hidden !== true)
+      .map(s => {
+        const description = s.description ? dim(s.description) : '';
+        return {
+          title: `${padEnd(generateStarterName(s), maxLength)}   ${description}`,
+          value: s.name,
+        };
+      }),
   ];
 }
 
+/**
+ * Generate the user-displayed name of the starter project
+ * @param starter the starter project to format
+ * @returns the formatted name
+ */
+function generateStarterName(starter: Starter): string {
+  // ensure that community packages are differentiated from those supported by Ionic/the Stencil team
+  return starter.isCommunity ? `${COMMUNITY_PREFIX} ${starter.name}` : starter.name;
```

Comment:
nit: could we put "[community]" after the starter name? I think it reads better to indicate what the starter is and then indicate whether or not it is community owned

Reply:
Yep! - done in https://github.com/ionic-team/create-stencil/pull/68/commits/785b3a108dec178103683341a56aef832255e607